src/main/java/Duke.java

```diff
@@ -1,10 +1,103 @@
+
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        Scanner UserInput = new Scanner(System.in);
+        int listCounter = 0;
+        Task[] myTasks = new Task[100];
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________\n" +
+                " Hello! I'm Duke\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n");
+
+        String command = UserInput.nextLine();  // scan user input
+
+        while (!command.equals("bye")) {     // prog doesnt end unless "bye"
+            if (command.equals("list")){        // show list
+                System.out.println("____________________________________________________________\n" +
```

Comment:
Maybe the println line can be placed at the start of the while loop instead of within each if-else statement since it will get executed no matter the case